src/main/java/org/codehaus/plexus/archiver/jar/module/ModuleDescriptorExtender.java

```diff
@@ -0,0 +1,146 @@
+/**
```

Comment:
I'm not quite happy with extending the module descriptor with byte code manipulation in the Plexus Archiver. What do you think about moving this class in some other component that could be shared with all plugins that would like to manipulate the  module descriptor. I could imagine that there will not be a lot of them so I'm ok for this to stay here.